.github/workflows/build-emscripten.yml

```diff
@@ -59,3 +59,18 @@ jobs:
     - name: Build
       if: success()
       uses: DiligentGraphics/github-action/build@v1
+
+    - name: Pack artifacts
+      if:  ${{ success() && (matrix.host_name == 'Linux') && (matrix.build_type == 'Release') }}
+      working-directory: ${{github.workspace}}
+      run: |
+        python ./DiligentSamples/BuildTools/Emscripten/pack-artifacts.py -s ${{ env.DILIGENT_BUILD_DIR }}/DiligentSamples -o ./BuildArtifacts
```

Comment:
Actually, let's rename the folder to be a but more Web-specific. Maybe `WebArtifacts`